meta-balena-common/recipes-containers/hostapp-update/files/hostapp-update

```diff
@@ -156,5 +187,15 @@ mv "$SYSROOT/counter.new" "$SYSROOT/counter"
 sync -f "$SYSROOT"
 
 if [ "$reboot" = 1 ]; then
-	reboot
```

Comment:
This will only run when executed from a running device, so add a conditional on API_ENDPOINT:
```
if [ -n "${API_ENDPOINT}" ]; then
    if ! call_supervisor_api "$PING_ENDPOINT"; then
        exit 1
    fi
    if ! call_supervisor_api "$REBOOT_ENDPOINT"; then
        exit 1
    fi
fi
```